src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,31 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+
+type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
+};
+
+type User = {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+};
```

Comment:
I'm not sure, if this should be in other file precisely for each special type file :)